src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -34,18 +47,45 @@ const AnnotationActivityMenu = ({ canDelete, canEdit, id, onDeleteConfirm, onEdi
         onDeleteConfirm();
     };
 
+    const handleMenuClose = () => {
+        setIsMenuOpen(false);
+    };
+
+    const handleMenuOpen = () => {
+        setIsMenuOpen(true);
+    };
+
     const tetherProps = {
         attachment: 'top right',
         className: 'bcs-AnnotationActivity-deleteConfirmationModal',
         constraints: [{ to: 'scrollParent', attachment: 'together' }],
         targetAttachment: 'bottom right',
     };
 
+    React.useEffect(() => {
+        const { current: isMounted } = isMountedRef;
```

Comment:
Even if I do that, the ref will have the value for `current` when the effect runs after first mount -- which will lead to `onMenuClose` being called because `!isConfirmingDelete && !isMenuOpen` is satisfied, so what I'm going for is essentially a "don't run the first time" effect